### PR TITLE
Added gantt/2-journeys-visualization

### PR DESCRIPTION
**Pull request by KamilKubik**
Gantt journeys visualization exercise

### CODE REVIEW

File: highcharts-api/highcharts-gantt/2-adventures-visualization/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Gantt exercise 2
+
+Open a chart from the `index.html` file with a default configuration showing a visualization of journeys. First, create a data structure, including the nested domestic/international flights shown in the attached gif. Based on your data, follow the next steps:
+
+1. Adjust the x-axis to show a whole week, and format the day labels to Day 1, Day 2, Day 3...
+2. Add a navigator, showing the whole series inside (from the first to the last point).
+3. Add a drag and drop functionality in the x dimension (except the last point that shows the international flight), and follow the below rules:
+- A drag should be available only in a range of the x-axis.
+- The dropped element should always position itself in the range of a particular day (as shown in a gif).
+- There should be a gray shadow showing a place, where the element will be dropped (as shown in a gif).
```
Comment (Vazonik) [inline highcharts-api/highcharts-gantt/2-adventures-visualization/README.md:10]: Missing indents
```suggestion
    - A drag should be available only in a range of the x-axis.
    - The dropped element should always position itself in the range of a particular day (as shown in a gif).
    - There should be a gray shadow showing a place, where the element will be dropped (as shown in a gif).
```

File: highcharts-api/highcharts-gantt/2-adventures-visualization/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+4. The last point, showing the international flight, should have a drilldown, showing two flights before reaching the final destination. Follow the below rules:
+- After the drilldown, the navigator should show the displayed points (as shown in a gif).
+- After the drillup, the navigator should be backed to the previous state, showing the whole series.
```
Comment (Vazonik) [inline highcharts-api/highcharts-gantt/2-adventures-visualization/README.md:13]:
```suggestion
    - After the drilldown, the navigator should show the displayed points (as shown in a gif).
    - After the drillup, the navigator should be backed to the previous state, showing the whole series.
```

**Review by Vazonik — CHANGES_REQUESTED** [review]
Nice! Only small changes to the markdown indentations would be good (you can automatically commit my suggestions).

I'm wondering if there is any way to better present the all data in the illustration so that I don't have to wait for the gif to expand. Maybe an image with an expanded Gantt with all visible points?

**Review by pawellysy — APPROVED** [review]
LGTM

**Review by mfilipiec — APPROVED** [review]
I can't notice anything more than David, so LGTM 😅

**Review by Vazonik — APPROVED** [review]
LGTM, thanks!

**Review by mfilipiec — APPROVED** [review]

**Pull request merged**